provisionerd/provisionerd.go

```diff
@@ -65,7 +65,7 @@ func New(clientDialer Dialer, opts *Options) *Server {
 		opts.UpdateInterval = 5 * time.Second
 	}
 	if opts.ForceCancelInterval == 0 {
```

Comment:
A thought/musing: Do we want to support disabling this by setting it to `0`? Or do we just ask users to raise it to a _really high number_ ™️?

(No change necessary here, the latter is fine.)

Reply:
Good question, Mathias. I wouldn't recommend setting it to infinite as it can be problematic for some cases (stuck forever). I guess that it implicates the latter option.

Reply:
That’s a very good point. So yeah, let’s not allow our users to create a deadlock with all provisioners waiting forever 😄